plonecli/cli.py

```diff
@@ -275,5 +275,21 @@ def build(context, clean):
     context.forward(run_buildout)
 
 
+@cli.command()
+def config():
+    """Configure mr.bob global settings"""
+    echo(
+        '\nRUN: mrbob bobtemplates.plone:configure',
+        fg='green',
+        reverse=True,
+    )
+    subprocess.call(
+        [
+            'mrbob',
+            'bobtemplates.plone:configure',
```

Comment:
Please add the mrbob template inside plonecli, not in bobtemplates.plone.
Because this is not related to bobtemplates.plone and we don't want to have templates there, which are not related to plone. That means, you have to create the strucure as it is in bobtemplates.plone, so that we have a bobtemplate in plonecli package. the call might look like "plonecli:configure_mrbob" and the folder config would be inside the plonecli package root. We would also add a Python file "configure_mrbob.py" which will contain our hooks.

Reply:
Okay, that makes sense